zk_toolbox/crates/zk_inception/src/consts.rs

```diff
@@ -9,6 +9,27 @@ pub const DEFAULT_PROOF_STORE_DIR: &str = "artifacts";
 pub const BELLMAN_CUDA_DIR: &str = "era-bellman-cuda";
 pub const L2_BASE_TOKEN_ADDRESS: &str = "0x000000000000000000000000000000000000800A";
 
+#[allow(non_upper_case_globals)]
+const kB: usize = 1024;
+
+/// Max payload size for consensus
+pub const MAX_PAYLOAD_SIZE: usize = 2_500_000;
+/// Max batch size for consensus
+/// Compute a default batch size, so operators are not caught out by the missing setting
+/// while we're still working on batch syncing. The batch interval is ~1 minute,
+/// so there will be ~60 blocks, and an Ethereum Merkle proof is ~1kB, but under high
+/// traffic there can be thousands of huge transactions that quickly fill up blocks
+/// and there could be more blocks in a batch then expected. We chose a generous
+/// limit so as not to prevent any legitimate batch from being transmitted.
+pub const MAX_BATCH_SIZE: usize = MAX_PAYLOAD_SIZE * 5000 + kB;
+/// Gossip dynamic inbound limit for consensus
+pub const GOSSIP_DYNAMIC_INBOUND_LIMIT: usize = 1;
```

Comment:
This will work assuming just 1 external node. If you want it future-proof set it to sth like 100